contrib/kind.sh

```diff
@@ -41,7 +42,11 @@ usage()
     echo "-i6 | --ipv6                 Enable IPv6. DEFAULT: IPv6 Disabled."
     echo "-wk | --num-workers          Number of worker nodes. DEFAULT: HA - 2 worker"
     echo "                             nodes and no HA - 0 worker nodes."
-    echo "-gm | --gateway-mode         Enable 'shared' or 'local' gateway mode. DEFAULT: local."
+    echo "-su | --system-update        Allow script to update system. Intended to allow"
```

Comment:
(inconsistency about "system update" vs "update system")

but also, the name is very non-indicative anyway...